official/recommendation/ncf_keras_benchmark.py

```diff
@@ -147,6 +147,14 @@ def benchmark_1_gpu_ctl_early_stop(self):
     FLAGS.early_stopping = True
     self._run_and_report_benchmark()
 
+  def benchmark_1_gpu_ctl_default_ds_run_eagerly_early_stop(self):
+    self._setup()
+    FLAGS.distribution_strategy = 'default'
```

Comment:
nice! did the default ds work out of the box?

Reply:
yes and I just renamed the test.